backend/src/main/java/com/springBoot/fruits_ecommerce/models/ErrorResponse.java

```diff
@@ -0,0 +1,100 @@
+package com.springBoot.fruits_ecommerce.models;
+
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Generated;
+
+/**
+ * ErrorResponse
+ */
+
+@Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2024-09-28T07:15:05.441995700+03:00[Asia/Riyadh]", comments = "Generator version: 7.8.0")
+public class ErrorResponse {
```

Comment:
Why do you have this "ErrorResponse" in project source code, it should be located in generated sources by OpenAPI plugin